.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es2017": true,
```

Comment:
Isn't ES9 ES2018?

Reply:
I didn't see an option for it here: https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments

Might be because no new global variables were introduced with es9.